src/components/PageLayout.js

```diff
@@ -200,12 +210,12 @@ const PageLayout = (props) => {
         <Box display="flex" flexDirection="column" height="100%">
           <Header>{footerFragment}</Header>
 
-          <Box position="relative" display="flex" flexGrow={1}>
+          <Box position="relative" display="flex" flexGrow={1} overflowY="auto">
```

Comment:
I'd like to get this approach approved with everyone first. It would prevent the IE11 footer being positioned on top of the content for long pages.

(which happens because we `height: 100%` all containers to allow the footer to `margin-top: auto` itself and therefore be positioned at the bottom of the screen for short pages. min-height: 100%/vh also do not solve this issue.)

Reply:
This should be achievable with flex unless I'm missing something?
What if we add `alignItems="stretch"` to the wrapping Box?

Reply:
This won't work. I don't believe in this case there's a fix which would satisfy all browsers. Feel free to play though.